config/locales/en.yml

```
@@ -2882,6 +2882,7 @@ en:
       new_features:
         header: "Read about new features and product updates."
         learn_about: "Learn more about all new features"
+        missing: "There are no highlight features presented yet."
```

_highlight_ and _presented_ sound awkward to me. Perhaps just:

```suggestion
        missing: "There are no highlighted features yet."
```